Remove pytorch quant docs since we are moving to torchao

jerryzh168 opened the pull request:
Stack from [ghstack](https://github.com/ezyang/ghstack) (oldest at bottom):
* __->__ #157766

Summary:
att

Test Plan:
doc page generated from CI

Reviewers:

Subscribers:

Tasks:

Tags:

cc @albanD

jerryzh168 committed:
Remove pytorch quant docs since we are moving to torchao

Summary:
att

Test Plan:
doc page generated from CI

Reviewers:

Subscribers:

Tasks:

Tags:

[ghstack-poisoned]

jerryzh168 committed:
Update on "Remove pytorch quant docs since we are moving to torchao"


Summary:
att

Test Plan:
doc page generated from CI

Reviewers:

Subscribers:

Tasks:

Tags:

[ghstack-poisoned]

jerryzh168 committed:
Update on "Remove pytorch quant docs since we are moving to torchao"


Summary:
att

Test Plan:
doc page generated from CI

Reviewers:

Subscribers:

Tasks:

Tags:

[ghstack-poisoned]

jerryzh168 committed:
Update on "Remove pytorch quant docs since we are moving to torchao"


Summary:
att

Test Plan:
doc page generated from CI

Reviewers:

Subscribers:

Tasks:

Tags:

cc albanD

[ghstack-poisoned]

albanD commented:
Yes skipping sanity check for removal is good!

albanD commented:
But removing docs for public API is not. You need to keep these and add a deprecation warning for them. Removing the landing page sounds good though.

jerryzh168 committed:
Update on "Remove pytorch quant docs since we are moving to torchao"


Summary:
att

Test Plan:
doc page generated from CI

Reviewers:

Subscribers:

Tasks:

Tags:

cc albanD

[ghstack-poisoned]

jerryzh168 committed:
Update on "Remove pytorch quant docs since we are moving to torchao"


Summary:
att

Test Plan:
doc page generated from CI

Reviewers:

Subscribers:

Tasks:

Tags:

cc albanD

[ghstack-poisoned]

jerryzh168 committed:
Update on "Remove pytorch quant docs since we are moving to torchao"


Summary:
att

Test Plan:
doc page generated from CI

Reviewers:

Subscribers:

Tasks:

Tags:

cc albanD

[ghstack-poisoned]

jerryzh168 committed:
Update on "Remove pytorch quant docs since we are moving to torchao"


Summary:
att

Test Plan:
doc page generated from CI

Reviewers:

Subscribers:

Tasks:

Tags:

cc albanD

[ghstack-poisoned]

jerryzh168 commented:
@pytorchbot merge
